### PR TITLE
Add listenAddrs to hotspot status.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: packages/http/src/models/Hotspot.ts [hunk 1]
```diff
@@ -25,7 +25,12 @@ export interface HTTPHotspotObject {
   block_added?: number
   geocode?: HTTPGeocodeObject
   address: string
-  status?: Status
```
Comment (danielcolinjames) [inline packages/http/src/models/Hotspot.ts:28]: should there be a different snake-cased-fields interface called HTTPStatusObject to match the way HTTPGeocodeObject is done? not a huge deal either way, and just trying to understand the code better (still getting up to speed with TypeScript and helium-js, so this comment is more for my own understanding)

**Review by danielcolinjames — APPROVED** [review]
looks good!

File: packages/http/src/models/Hotspot.ts [hunk 2]
```diff
@@ -115,7 +121,12 @@ export default class Hotspot extends DataModel {
     this.lng = hotspot.lng
     this.lat = hotspot.lat
     this.block = hotspot.block
-    this.status = hotspot.status
+    this.status = {
+      gps: hotspot.status?.gps || '',
+      height: hotspot.status?.height || 0,
```
Comment (tyler-whitman) [inline packages/http/src/models/Hotspot.ts:126]: Does the API return 0 or null? Might be good to tell the difference between one that is undefined and one that is actually 0?

File: packages/http/src/models/Hotspot.ts (hunk 1 above)
Comment (tyler-whitman) [reply packages/http/src/models/Hotspot.ts:28]: Or should we just update the type that is already there for Status?

Comment (matthewcarlreetz) [reply packages/http/src/models/Hotspot.ts:28]: The Status that was there is the camelCased status. For the HTTP object we need snake case. We could create an interface `HTTPStatusObject`, but seeing as though it is only used in this context I didn't. If we ever wanted to reuse it, then we definitely would want to create a separate interface.

File: packages/http/src/models/Hotspot.ts (hunk 2 above)
Comment (matthewcarlreetz) [reply packages/http/src/models/Hotspot.ts:126]: I'm not certain, but I think in this scenario null and 0 have the same meaning (sync hasn't started)

File: packages/http/src/models/Hotspot.ts (hunk 1 above)
Comment (tyler-whitman) [reply packages/http/src/models/Hotspot.ts:28]: Ah its only for the HTTP type, ok this is fine. Should we update that status object in the hotspot to include the new camel case parameters? I think its missing `listenAddrs` ?

Comment (tyler-whitman) [reply packages/http/src/models/Hotspot.ts:28]: Wait nevermind its there lol

**Review by tyler-whitman — APPROVED** [review]

File: packages/http/src/models/Hotspot.ts [hunk 3]
```diff
@@ -60,6 +65,7 @@ interface Status {
   gps: string
   height: number
   online: string
+  listenAddrs: string[]
```
Comment (matthewcarlreetz) [inline packages/http/src/models/Hotspot.ts:68]: @tyler-whitman you mean this? ^

File: packages/http/src/models/Hotspot.ts (hunk 2 above)
Comment (tyler-whitman) [reply packages/http/src/models/Hotspot.ts:126]: Yeah I think it should be ok

File: packages/http/src/models/Hotspot.ts (hunk 3 above)
Comment (tyler-whitman) [reply packages/http/src/models/Hotspot.ts:68]: Haha yeah, missed it sorry.

**Review by allenan — APPROVED** [review]

**Pull request merged**